### ISSUE TITLE
Surface Sentry SDK client status reports in Spotlight

### ISSUE THREAD

**Issue by BYK** [getsentry/spotlight#477]
It would be great to surface SDK client reports such as events getting dropped due to size etc.

**Comment by Shubhdeep12** [comment]
how can we get this information?

**Comment by BYK** [comment]
@Shubhdeep12 It's just a different type of event: https://develop.sentry.dev/sdk/telemetry/client-reports/

**Comment by Shubhdeep12** [comment]
Okay. 
I never read about this one. 

Also, is there a way I can trigger this event manually? 
Looks like there is no built-in function to trigger this.

**Comment by BYK** [comment]
Yeah, I agree it's tricky to trigger. I think easist would be to install a `beforeSend` hook and keep dropping events there to trigger a report. I saw that's one of the categories.
